docs/checks.md

```diff
@@ -317,9 +317,11 @@ low score is therefore not a definitive indication that the project is at risk.
 Risk: `Medium` (possible vulnerabilities in code)
 
 This check tries to determine if the project uses
-[fuzzing](https://owasp.org/www-community/Fuzzing) by checking if the repository
-name is included in the [OSS-Fuzz](https://github.com/google/oss-fuzz) project
-list.
+[fuzzing](https://owasp.org/www-community/Fuzzing). Currently, scorecard checks:
+1. if the repository name is included in the [OSS-Fuzz](https://github.com/google/oss-fuzz) project
+list;
+2. if [ClusterFuzzLite](https://google.github.io/clusterfuzzlite/) is deployed in the repository;
+3. if there are user-defined language-specified fuzzing functions (now supports [Go fuzzing](https://go.dev/doc/fuzz/)) in the repository.
```

Comment:
It's not clear from this parenthetical phrase whether Go fuzzing is the only language-specified user-defined fuzzing feature supported, or whether the "now" implies it's being added to an existing list. Can you clarify? For example, "if there are user-defined language-specified fuzzing functions in the repository (currently supports X, Y, and Z)."